directlfq/dashboard_parts.py

```diff
@@ -327,6 +327,8 @@ def run_pipeline(self, *args):
         file_of_proteins_for_normalization = None if self.protein_subset_for_normalization_file.value == '' else self.protein_subset_for_normalization_file.value
         num_cores = None if self.num_cores_vals.value == -1 else self.num_cores_vals.value
         yaml_filt_dict_path = None if self.yaml_filt_dict_path.value == '' else self.yaml_filt_dict_path.value
+        if type(additional_headers) == str: #the user will enter a string with semicolon separated values
```

Comment:
can this be any other type?

Reply:
any user input will be converted to a string. If there is no user input, there will be None

Reply:
so this could be `if additional_headers is not None:`?